prefect_twitter/media.py

```diff
@@ -0,0 +1,71 @@
+"""This is a module for interacting with Twitter media"""
+
+from functools import partial
+from pathlib import Path
+from typing import TYPE_CHECKING, Union
+
+from anyio import to_thread
+from prefect import get_run_logger, task
+
+if TYPE_CHECKING:
+    from io import IOBase
+
+    from prefect_twitter import TwitterCredentials
+
+
+@task
+async def media_upload(
+    filename: Union[Path, str],
+    twitter_credentials: "TwitterCredentials",
+    file: "IOBase" = None,
+    chunked: bool = None,
```

Comment:
Boolean args should default to either True or False to avoid confusion.
```suggestion
    chunked: bool = False,
```